docs/source/newsfragments/4517.feature.rst

```diff
@@ -0,0 +1 @@
+:class:`.HierarchyObjectBase` and its inheritants now support :meth:`_get` for a dictionary-style handle retrieval, which returns ``None`` on failure instead of raising an exception to aid with optional signal discovery.
```

Comment:
```suggestion
:class:`.HierarchyObjectBase` and its inheritants now support :meth:`~cocotb.handle.GPIDiscovery._get` for a dictionary-style handle retrieval, which returns ``None`` on failure instead of raising an exception to aid with optional signal discovery.
```

Reply:
no this is wrong. _get is a method of HierarchyObjectBase